ivy/utils/binaries.py

```diff
@@ -9,8 +9,9 @@
 def _get_paths_from_binaries(binaries, root_dir=""):
```

Comment:
The changes made to this helper would also need to be replicated [here](https://github.com/unifyai/ivy/pull/28150/files#diff-60f61ab7a8d1910d86d9fda2261620314edcae5894d5aaa236b821c7256badd7R27)